README.md

```diff
@@ -1,2 +1,198 @@
-# math-magicians
-create-react-app exercise.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h2><b>Create React App: exercise</b></h2>
```

Comment:
- [ OPTIONAL ] Great job setting up your project! However, I noticed that the project name provided here is incorrect. I suggest that you update it with the correct project name. Providing the correct project name in the README is important because it helps to ensure that users can easily identify and search for your project, which reflects attention to detail and professionalism. :wink:

Reply:
I will take care of it before the next submission.